### PR TITLE
upgrade to ForgeGradle 5

### PR DESCRIPTION
**Pull request by 123FLO321**
- needs testing, but it appears to work fine so far

### CODE REVIEW

**Comment by ALongStringOfNumbers** [comment]
Could you update Gradle to 7.6 instead of 7.5 (to match with the main project) via the command line option `./gradlew wrapper --gradle-version 7.6`

**Comment by 123FLO321** [comment]
@ALongStringOfNumbers updated

File: build.gradle [hunk 1]
```diff
@@ -1,119 +1,376 @@
...
-    //runtime "curse.maven:magic-bees-65764:2566282"
-    //runtime "curse.maven:gendustry-70492:2516215"
-    //runtime "curse.maven:bdlib-70496:2518031"
-    //runtime "curse.maven:career-bees-281791:2534481"
-    //runtime "curse.maven:retrobees-300139:2934369"
-    //runtime "curse.maven:the-twilight-forest-227639:3051450"
-    //runtime "curse.maven:thaumcraft-223628:2629023"
-    //runtime "curse.maven:baubles-227083:2518667"
-    //runtime "curse.maven:biomes-o-plenty-220318:2715506"
-    //runtime "curse.maven:extra-utilities-225561:2678374"
```
Comment (ALongStringOfNumbers) [inline build.gradle:85]: Please keep these commented out deps

File: gradle.properties [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+mixingradle_version = 0.7-SNAPSHOT
+mixinbooter_version = 7.0
+mixin_annotations_version = 0.8.5
+
+## Tests
+junit_version = 5.9.1
+hamcrest_version = 2.2
+
+## Compile-Time Dependencies
+lombok_version = 1.18.16
```
Comment (ALongStringOfNumbers) [inline gradle.properties:55]: I don't believe lombok is needed here. Not seeing it used in build.gradle

**Review by DStrand1 — APPROVED** [review]

**Pull request merged**